lib/internal-rules/internal-no-invalid-meta.js

```diff
@@ -191,35 +175,13 @@ module.exports = {
                 }
             },
 
-            CallExpression(node) {
```

Comment:
This is an internal rule. We were using this logic to validate that rules had valid `fixable` metadata, but the [require-meta-fixable](https://github.com/not-an-aardvark/eslint-plugin-eslint-plugin/blob/master/docs/rules/require-meta-fixable.md) rule from the plugin takes care of this.